README.md

```diff
@@ -1,4 +1,11 @@
-# Prometheus
+<h1 align="center" style="border-bottom: none">
+    <a href="//prometheus.io"><img alt="Prometheus" src="/web/ui/react-app/src/images/logo-prometheus-orange.png" width="80"></a><br>Prometheus
+</h1>
+
+<p align="center">Visit <a href="https://prometheus.io" blank="_new">prometheus.io</a> for the full documentation,
```

Comment:
I couldn't find any info on a `blank` attribute for `a` tags (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). I guess this should be `target="_blank"` instead?

Maybe also use the `//` scheme here as above for consistency? Or just set all of them to `https`, since we don't really do `http` anymore anyway.

Reply:
oh no my bad, i was not focus, fixing!